INSTALL.rst

```diff
@@ -26,19 +26,18 @@ In order to execute the tests and compare them against precalculated results,
 you will additionally need:
 
 * Python (version >= 2.6) with NumPy
-  
-* The following Slater-Koster data sets: hyb-0-2, mio-1-1, pbc-0-3, rare-0-2
-  which are available from `dftb.org <http://www.dftb.org>`_.
 
 
 Obtaining the source
 ====================
 
 The source code can be downloaded from the `DFTB+ homepage
-<http://www.dftbplus.org>`_ or directly from the `public git repository
-<https://github.com/dftbplus/dftbplus>`_. (The tagged revisions correspond to
-stable releases, while the master branch contains the latest development
-version.)
+<http://www.dftbplus.org>`_. Alternatively you can clone the `public git
+repository <https://github.com/dftbplus/dftbplus>`_. (The tagged revisions
+correspond to stable releases, while the master branch contains the latest
+development version.) Note: the project uses git-submodules, which must be
+additionally downloaded using the ``git submodule update`` command after the
```

Comment:
git submodule update --recursive --remote
might be better if  git 1.8.2 or above